tslint.js

```diff
@@ -5,6 +5,12 @@ module.exports = {
         path.join(path.dirname(require.resolve('tslint-eslint-rules')), 'dist/rules')
     ],
     rules: {
+        "ban": [
+            true,
+            "eval",
+            { "name": ["it", "only"], "message": "don't focus tests" },
```

Comment:
we should add in "fit" and "fdescribe" to the angular 2 stuff we work on

Reply:
updated